duckduckgo_search/duckduckgo_search_async.py

```diff
@@ -25,18 +26,22 @@
 class AsyncDDGS(metaclass=GoogleDocstringInheritanceMeta):
     """DuckDuckgo_search async class to get search results from duckduckgo.com."""
 
-    def __init__(self, headers=None, proxies=None, timeout=10) -> None:
+    def __init__(self, headers=None, proxies=None, timeout=10, verify=True) -> None:
         """Initialize the AsyncDDGS object.
 
         Args:
             headers (dict, optional): Dictionary of headers for the HTTP client. Defaults to None.
             proxies (Union[dict, str], optional): Proxies for the HTTP client (can be dict or str). Defaults to None.
             timeout (int, optional): Timeout value for the HTTP client. Defaults to 10.
+            verify  (bool, optional): Whether or not to verify HTTPS requests. Defaults to True.
         """
         self.proxies = proxies if proxies and isinstance(proxies, dict) else {"http": proxies, "https": proxies}
         self._asession = requests.AsyncSession(
-            headers=headers, proxies=self.proxies, timeout=timeout, impersonate=_random_browser()
+            headers=headers, proxies=self.proxies, timeout=timeout, impersonate=_random_browser(), verify=verify
         )
+        logger.warning("AsyncDDGS verify flag set to False. HTTPS requests "
```

Comment:
1) In this case, every time the class is initialized, this message will appear.
The message should be shown if verify=False.
2) I think it is better to use warnings.warn instead of logging.
https://docs.python.org/3/library/warnings.html
```python3
Repetitions of a particular warning for the same source location are typically suppressed.
```

Reply:
1. You're absolutely right; sorry for the mistake. I fixed this and added test cases to make sure the warning isn't raised when it shouldn't be.
2. I agree. I'm using the warnings library now.